sections/collage.liquid

```diff
@@ -27,7 +27,7 @@
         <div class="collage__item collage__item--{{ block.type }} collage__item--{{ section.settings.desktop_layout }}" {{ block.shopify_attributes }}>
           {%- case block.type -%}
             {%- when 'image' -%}
-              <div class="collage-card color-{{ settings.card_color_scheme }}{% if block.settings.image_padding %} collage-card-spacing{% endif %}">
+              <div class="collage-card {% if section.settings.card_styles == '' %}global-media-settings{% else %}card-wrapper product-card-wrapper color-{{ settings.card_color_scheme }}{% endif %}">
```

Comment:
Following my suggestion above, I would do:
```suggestion
              <div class="collage-card {% if section.settings.card_styles == 'none' %}global-media-settings{% else %}card-wrapper product-card-wrapper color-{{ settings.card_color_scheme }}{% endif %}">
```
Also, another question: is there any reason for us to output `product-card-wrapper` directly here and not the dynamic value of `{{ section.settings.card_styles }}`? _(check comment below)_